<launch>
  <group ns="slam">
    <node pkg="ndt_gslam" type="graph_slam2d" name="graph_slam2d" respawn="true" output="screen">
      <!-- <param name="tf_prefix" value="p3dx_0"/> -->
      <param name="robot_base_frame_id" value="base_footprint"/>
      <!-- incomming odometry  tf frame_id without tf_prefix -->
      <param name="odom_farme_id" value="odom_combined"/>
      <!-- calculated map frame_id name -->
      <param name="map_farme_id" value="map"/>

      <remap from="/scan" to="/base_scan" />
      <!-- ODOM, POSE, default:NON -->
      <param name="subscribe_mode" value="NON"/>

       <param name="scanmatch_window_radius" type="double" value="30.0"/>
       <param name="node_gen_distance" type="double" value="2.0"/>
       <param name="loop_max_distance" type="double" value="20.0"/>
       <param name="loop_min_distance" type="double" value="20.0"/>
       <param name="loop_score_threshold" type="double" value="0.6"/>

       <param name="serialize_graph" type="bool" value="true"/>
    </node>
  </group>
</launch>